### ISSUE TITLE
New (And Better Integrated) ICONS!!! And 1 or 2 More Glitches! <3

### ISSUE THREAD

**Issue by Something-Went-Terribly-Wrong** [esapee/Essence-Armory#5]
I made ya some new icons for the vambraces, pauldrons, cloak **(for visual purposes only)**, and a scroll (for a "Essence Armorer" title)! The crafting recipe for the cloak should be one essence infused string and blocks of essence shards. And the recipe for the scroll should be a normal scroll surrounded by essence plated stars >:D
You don't need to use them, but I'd like to help you regardless <3
Oh and while I'm at it, nei says you can't craft the essence blade (I haven't tried to craft it so idk if its just a nei glitch) and nei doesnt give the recipe for flux infusion (not sure if I'm missing the compatible mod but I have no clue what flux infusion would do regardless)
![imported piskel](https://cloud.githubusercontent.com/assets/19368215/16906752/97025bec-4c8a-11e6-91fe-0a523b84ce35.png)
![imported piskel 1](https://cloud.githubusercontent.com/assets/19368215/16907100/5c6e67ce-4c8e-11e6-9623-782cb3995f9c.png)
![imported piskel 2](https://cloud.githubusercontent.com/assets/19368215/16907613/fa3f0efa-4c92-11e6-9ba5-29bd195e8f3b.png)
![imported piskel 3](https://cloud.githubusercontent.com/assets/19368215/16908970/4e7efdea-4c9d-11e6-82bc-6d37d49f8646.png)
Along with the next photo for the cloak model
![cloak](https://cloud.githubusercontent.com/assets/19368215/16908886/bfd111dc-4c9c-11e6-81ed-9c471bed75e5.png)

**Comment by Something-Went-Terribly-Wrong** [comment]
Oh man, that took me 4 hours! #Worth <3

**Comment by murapix** [comment]
I should probably get you on our discord, these textures are pretty good, and it would make bug reports easier.

**Comment by Something-Went-Terribly-Wrong** [comment]
ikr, I already have a discord account so all you have to do is invite me or give me a link :D
My discord account is "SomethingWentTerriblyWrong"

**Comment by Something-Went-Terribly-Wrong** [comment]
My discord account is "SomethingWentTerriblyWrong"

**Comment by Something-Went-Terribly-Wrong** [comment]
Hey @murapix you there?

**Comment by Something-Went-Terribly-Wrong** [comment]
My email is "alexgill12354@gmail.com" if you want to contact me personally.

**Comment by murapix** [comment]
Sorry for going silent for a while, irl stuff kinda got in the way. I unfortunately don't have permissions to add you to the team's discord channel, but I will want you around when I get started on recreating the mod.

As for the textures, thanks for them, but I've kinda moved on from the 1.7.10 version, and adding more content to a largely discontinued mod isn't really on my priorities list.

**Comment by Something-Went-Terribly-Wrong** [comment]
Recreating the mod? I would defiantly love to help on that front, contact me whenever, I'm always willing to help! <3

**Comment by Something-Went-Terribly-Wrong** [comment]
Well I guess I'll have to do a bit of 7-Zip magic to update the mod myself ;D

**Comment by Something-Went-Terribly-Wrong** [comment]
Okay never mind, I couldn't find an decompiled version of 1.0.0-c so I 7-ziped it's contents into a folder, then proceeded to take the pauldrons .class file and then make a .java file of that code (note I decompiled that single file first) After all of that, I changed all of the "pauldrons" to "cloak" and tried compiling it into a class file using javac. F*CK JAVAC! No matter what I tried, Javac would not cooperate with me so I couldn't make the .class file for the cloak. The same thing applied for the title, but I was able to change the pauldron's and vambrace's texture. But its kinda sad that I couldn't add the cloak (which looks so cool, I temp replaced the original .png for the model to test >:D) or the "Essence Armorer" title. Regardless I still love your mod and dedication <3

**Comment by murapix** [comment]
Unfortunately, there's a lot more to making an item than just creating another java file with by copying files from the source code. I looked into the cloak and title, and both of them require at least twice the code that a plain item does, possibly more for the title - I couldn't completely figure that one out in the time I looked at it. But yes, screw javac. If you want to do any sort of coding, or editing mods, you'll need to set up a modding workspace in an actual IDE.
